pennylane/qchem/integrals.py

```diff
@@ -56,10 +56,15 @@ def primitive_norm(l, alpha):
     array([1.79444183])
     """
     lx, ly, lz = l
+
+    fac_x = fac2(2 * lx - 1) if lx > 0 else 1
+    fac_y = fac2(2 * ly - 1) if ly > 0 else 1
+    fac_z = fac2(2 * lz - 1) if lz > 0 else 1
```

Comment:
Since Its the same logic repeated everywhere, you could just define a private function at the top of the file and call it in both `primitive norm and contracted norm`:

```
def _fact2(l): return fac2(2 * l - 1) if l > 0 else 1 
```

Reply:
Added.